app/services/hyrax/listeners/file_metadata_listener.rb

```diff
@@ -12,8 +12,20 @@ class FileMetadataListener
       # @param [Dry::Events::Event] event
       # @return [void]
       def on_file_characterized(event)
-        CreateDerivativesJob
-          .perform_later(event[:file_set], event[:file_id], event[:path_hint])
+        file_set = event[:file_set]
+
+        case file_set
+        when ActiveFedora::Base # ActiveFedora
+          CreateDerivativesJob
+            .perform_later(file_set, event[:file_id], event[:path_hint])
+        else
+          # With valkyrie, file.characterized event is also published for derivatives.
```

Comment:
i think we don't want this to be true!

the valkyrie code path shouldn't be calling characterization on files that don't require characterization, right? i think this filter should go below around https://github.com/samvera/hyrax/pull/6180/files#diff-c1500de3cd457700a4445ea1b341164b5a822aa64a086d85e12eb9adfdfeba79R54, or else in the `ValkyrieCharacterizationService`?

Reply:
>  But in UCSD dams, currently we have file characterization for all files, including derivatives.

that seems okay. if downstream applications want to characterize under different conditions, it's easy to subscribe a new listener with the desired behavior.